tests/OmniSharp.Cake.Tests/FindSymbolsFacts.cs

```diff
@@ -23,25 +24,41 @@ public FindSymbolsFacts(ITestOutputHelper testOutput) : base(testOutput)
         [Fact]
         public async Task ShouldFindSymbolsInCakeProjects()
         {
-            var symbols = await FindSymbols("CakeProject");
+            var symbols = await FindSymbols("CakeProject", null, null);
             Assert.NotEmpty(symbols.QuickFixes);
         }
 
+        [Fact]
+        public async Task ShouldNotFindSymbolsInCakeProjectsDueToEmptyFilter()
+        {
+            var symbols = await FindSymbols("CakeProject", 1, 0);
```

Comment:
Nit: can you use named args for the integer consts/nulls?

Reply:
updated